server/startup.js

```diff
@@ -0,0 +1,31 @@
+Meteor.startup(function () {
+    if (Products.find().count() == 0) {
+        console.log('Loading products to the collection...');
+
+        _.each(JSON.parse(Assets.getText('products.json')), function (product) {
+            Products.insert(product);
+        });
+
+        console.log('Loaded ' + Products.find().count() + ' products');
```

Comment:
I works with mup. I didn't have a chance to test with meteor.com, but it should work. Takes less than 3 seconds to complete.